third_party/ros/roslaunch/deps.py.tpl

```diff
@@ -1,2 +1,2 @@
-ROSMASTER_PATH = 'external/ros_comm/rosmaster'
+ROSMASTER_PATH = 'external/_main~non_module_dependencies~ros_comm/rosmaster'
```

Comment:
OK, it's about time to auto-generated this path instead of hard-coding it. The same for the change in third_party/ros/roslaunch/roscore.xml.

Reply:
I'll try to schedule some time for this in the coming days.

Reply:
Good idea as the name `_main~non_module_dependencies` will change depending on what they called it in the `MODULE` file.

Reply:
https://github.com/mvukov/rules_ros/pull/18

Done